main.js

```diff
@@ -153,6 +156,7 @@ function createJitsiMeetWindow() {
         minHeight: 600,
         show: false,
         webPreferences: {
+            experimentalFeatures: true, // Insertable streams, for E2EE.
```

Comment:
can we use the more specific RTCInsertableStreams field trial? the command line flag for that is --force-fieldtrials=RTCInsertableStreams/Enabled

Reply:
Never tried that on Electron. Any reason why this might be undesirable?

Reply:
not sure what other experimental features we enable by this. OTOH we don't use them...

Reply:
this is equivalent to "experimental web platform features" I'll check the field trial thing.

Reply:
Looks like that one is not supported: https://www.electronjs.org/docs/api/command-line-switches